### ISSUE TITLE
SoluteControl does not work on iOS VoiceOver

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/membrane-transport#394]
From https://github.com/phetsims/qa/issues/1280#issuecomment-3155777638.

This control does not work on iOS VoiceOver. The tab stop goes to the entire control. But since it is not a "slider", usual swipe gestures do not work. If we need to support iOS VoiceOver, we probably need a different design.

**Comment by brettfiedler** [comment]
Noting we run into the same issue with Android/TalkBack (testd on my Pixel 9a)
